_includes/v21.1/app/txn-sample.cs

```diff
@@ -11,11 +10,14 @@ class MainClass
     static void Main(string[] args)
     {
       var connStringBuilder = new NpgsqlConnectionStringBuilder();
-      connStringBuilder.Host = "localhost";
+      connStringBuilder.Host = "<host-name>";
```

Comment:
We don't use this included code sample file any more with this PR so we should remove it.